packages/core/src/components/sections/Newsletter/Newsletter.tsx

```diff
@@ -61,13 +79,15 @@ const Newsletter = function Newsletter({
   displayNameInput,
   nameInputLabel,
   subscribeButtonLabel,
+  subscribeButtonLoadingLabel,
   card,
   toastSubscribe,
   toastSubscribeError,
+  colorVariant,
   ...otherProps
 }: NewsletterProps) {
   return (
-    <Section className={`${styles.section} section-newsletter`}>
+    <Section className={`${styles.section} section-newsletter layout__section`}>
```

Comment:
Why did you add this class?

Reply:
I just replicated this class [based on the changes](https://github.com/vtex/faststore/pull/1902/files#diff-763c1ca044e8603b0d663a7764b60255ac3cfaf2a87f841f17846e6aad6d5723) that @renatamottam had already applied before.

Reply:
All our page sections have this class. It adds our default vertical margins.